libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts

```diff
@@ -231,4 +237,13 @@ export class DrivingLicenseSubmissionService extends BaseTemplateApiService {
       }
     }
   }
+
+  async glassesCheck({ auth }: TemplateApiModuleActionProps): Promise<boolean> {
+    const licences: DriverLicenseWithoutImages[] =
+      await this.drivingLicenseService.getAllDriverLicenses(auth.authorization)
+    const hasGlasses: boolean = licences.some((license) => {
+      return !!license.comments?.some((comment) => comment.nr?.includes('01.'))
```

Comment:
Are there multiple codes for glasses? Why isn't this narrower?

Reply:
There are multiple codes for vision impairment. They're all prefixed with 01.

Reply:
Could you put it in a comment above. It would be nice for context, if this needs to be changed or altered later.